crates/bevy_pbr/src/deferred/pbr_deferred_functions.wgsl

```diff
@@ -53,6 +53,24 @@ fn deferred_gbuffer_from_pbr_input(in: PbrInput) -> vec4<u32> {
     } else {
         base_color_srgb = pow(in.material.base_color.rgb, vec3(1.0 / 2.2));
     }
+#ifdef LIGHTMAP
+    if (any(in.lightmap_light > vec3(0.0f))) {
```

Comment:
Considering the ifdef is already here, I wonder if it's better to not branch here for to keep the threads for diverging based on light map data. Also without adding to the emissive this means that fragments were the light map is black would let through emissive and then potentially have a hard switch to no emissive.